tests/test_integrated_channels/test_integrated_channel/test_transmitters/test_course_metadata.py

```diff
@@ -25,6 +25,8 @@ class TestCourseTransmitter(unittest.TestCase):
     def setUp(self):
         super(TestCourseTransmitter, self).setUp()
         enterprise_customer = factories.EnterpriseCustomerFactory(name='Starfleet Academy')
+        # We need some non-abstract configuration for these things to work,
+        # so it's okay for it be to any arbitrary channel. We randomly choose SAPSF.
```

Comment:
@UmanShahzad Ack! There's a typo here: "be to" should be "to be".

Be to or not be to, that is the question! (Sorry, couldn't resist 😎)

Reply:
:sunglasses: :sunglasses: :sunglasses: